turtle_graphics/drawing_turtle_graphics-Style.js

```diff
@@ -0,0 +1,205 @@
+class Turtle {
+    constructor(x, y) {
+        this.presentPosition = [x, y]; // the present position
+        this.allPointsArr = []; // all the points in this array
+        this.allPointsArr.push(this.presentPosition);
+        this.sequentialPoints = []; // use for print()
+        let temp = [x, y];
+        temp.push(false);
+        this.sequentialPoints.push(temp);
+        // 1: east; 2: south; 3: west; 4: north
+        this.direction = 1;
+        // east:[1, 0]; south:[0,1]; west:[-1,0]; north:[0,-1];
+        this.directionArr = [1, 0];
+        this.mapArr = [];
+        this.maxX = x;
+        this.minX = x;
+        this.maxY = y;
+        this.minY = y;
+        return this;
+    }
+    right() {
+        this.direction++;
+        switch (this.direction) {
+            case 2:
+                this.directionArr = [0, -1];
+                break;
+            case 3:
+                this.directionArr = [-1, 0];
+                break;
+            case 4:
+                this.directionArr = [0, 1];
+                break;
+            default:
+                this.direction = 1;
+                this.directionArr = [1, 0];
+                break;
+        }
+        return this;
+    }
+    left() {
+        this.direction--;
+        switch (this.direction) {
+            case 1:
+                this.directionArr = [1, 0];
+                break;
+            case 2:
+                this.directionArr = [0, -1];
+                break;
+            case 3:
+                this.directionArr = [-1, 0];
+                break;
+            default:
+                this.direction = 4;
+                this.directionArr = [0, 1];
+                break;
+        }
+        return this;
+    }
+    forward(steps) {
+        for (let i = 0; i < steps; i++) {
+            let temp = [this.presentPosition[0] + this.directionArr[0], this.presentPosition[1] + this.directionArr[1]];
+            this.presentPosition = [temp[0], temp[1]];
+            this.allPointsArr.push(this.presentPosition);
+            temp.push(false);
+            this.sequentialPoints.push(temp);
+        }
+        return this;
+    }
+    allPoints() {
+        // this.getMaxAndMin();
+        // let valX = 0,
+        //     valY = 0;
+        // if (this.minX < 0) {
+        //     valX = 1 - this.minX;
+        // }
+        // if (this.minY < 0) {
+        //     valY = 1 - this.minY;
+        // }
+        // let result = [];
+        // this.allPointsArr.forEach(element => {
+        //     result.push([element[0] + valX, element[1] + valY]);
+        // });
+        // return result;
```

Comment:
Remove unnecessary commented out lines of code such as these and lines `187` and `203` when making a PR.